src/arch/ofi/machine.C

```diff
@@ -195,17 +315,19 @@ static inline int process_completion_queue();
  * OFI RMA Header
  * Message sent by sender to receiver during RMA Read of long messages.
  *  - nodeNo: Target node number
- *  - src_msg: Address of source msg; Sent back as part of OFIRmaAck
+ *  - src_msg: Address or offset from registered source address
  *  - len: Length of message
  *  - key: Remote key
  *  - mr: Address of memory region; Sent back as part of OFIRmaAck
+ *  - orig_msg: actual address of source message; Sent back as part of OFIRmaAck
  */
 typedef struct OFIRmaHeader {
     uint64_t src_msg;
     uint64_t len;
     uint64_t key;
     uint64_t mr;
     int      nodeNo;
+    uint64_t orig_msg;
```

Comment:
Could swap nodeNo and orig_msg declaration order and potentially save 32 bits on the size of OFIRmaHeader from padding/alignment

Reply:
good idea. swapped in latest commit